src/ActionSheet/index.tsx

```diff
@@ -25,6 +26,7 @@ interface Props {
   readonly pointerEvents?: ViewProps['pointerEvents'];
 }
 
+const FLEX_ONE_STYLE: ViewStyle = { flex: 1 };
```

Comment:
This should be probably set on `styles`, at the bottom of this file

Reply:
Fixed, thanks!